### DISCUSSION TITLE
Strange Paradise

### DISCUSSION THREAD

**Discussion by what-a** [iptv-org/community#746] [Show and tell]
**Channel Name:** Strange Paradise
**Channel ID:** StrangeParadise.us
**Country:** USA
**Language:** English
**Website:** www.filmon.com/channel/strange-paradise
**Stream URL (optional):** https://edge-1282-ch-gv.filmon.com/live/3869.high.stream/playlist.m3u8?id=035bca1a71b11fce016d28acd3dbea51cbc96ddfe5b9ed3f8fe3286df6a738d31afee2380e236e0af112122a5c1df3a0203a51d8b3e4a4f9a7437cd070b7dd9e67d18821290ba7186e8e69608b09fcb4518dc9a9d0e9b0356147159e1bdd55df1aeb8ab0d8a864968f3e2a43f0c1553de885a6ea9417fc787fa604d1328e5e13e87c40d649b527981b73fd755f324b1b99c237cde5103a99
**Notes (optional):** As you guessed, the above link only worked for a few moments.

**Comment by BellezaEmporium** [comment]
I *think* we can use streamlink for that.

http://free.fullspeed.tv/iptv-query?streaming-ip=http://www.filmon.com/channel/strange-paradise

There's also the filmon api that we can use : https://www.filmon.com/api-v2/channel/3869, simply grab either the low or high link that's being given.

**Comment by freearhey** [comment]
https://edge-1282-ch-gv.filmon.com/live/3869.high.stream/playlist.m3u8?id=035bca1a71b11fce016d28acd3dbea51cbc96ddfe5b9ed3f8fe3286df6a738d31afee2380e236e0af112122a5c1df3a0203a51d8b3e4a4f9a7437cd070b7dd9e67d18821290ba7186e8e69608b09fcb4518dc9a9d0e9b0356147159e1bdd55df1aeb8ab0d8a864968f3e2a43f0c1553de885a6ea9417fc787fa604d1328e5e13e87c40d649b527981b73fd755f324b1b99c237cde5103a99 (offline)

![image](https://github.com/user-attachments/assets/d8d18a4a-7e74-4665-b135-dc80de9df64d)
